README.md

```diff
@@ -1,6 +1,6 @@
 ![Elixir Companies](https://user-images.githubusercontent.com/73386/33328317-e6e58c6e-d416-11e7-9a16-b60700db0a51.png)
 
-[![Build Status](https://travis-ci.org/beam-community/elixir-companies.svg?branch=master)](https://travis-ci.org/beam-community/elixir-companies)
+![](https://github.com/beam-community/elixir-companies/workflows/.github/workflows/elixir.yml/badge.svg?branch=master)
```

Comment:
In the preview it is not displayed properly. Is it just something temporary?

Reply:
Think so, the action has not yet run in the master branch of `beam-community/elixir-companies`. 

I got the badge url from https://help.github.com/en/actions/automating-your-workflow-with-github-actions/configuring-a-workflow#example-using-a-workflow-file-path